tensorflow/core/api_def/python_api/api_def_SamplingDataset.pbtxt

```diff
@@ -0,0 +1,4 @@
+op {
```

Comment:
get rid of this file and instead add the `visibility: HIDDEN` attribute to `tensorflow/core/api_def/base_api/api_def_SamplingDataset.pbtxt`

Reply:
fixed in https://github.com/tensorflow/tensorflow/pull/25384/commits/fa11069c3e85ffbcae19994bdba09c30eee0a6de#diff-10942c7d6fdf513b324aefbdcc44cedaR3